iOS/SideDish-team24/SideDish-team24/Extension/UIColorExtension.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+import UIKit
+
+extension UIColor {
+    static let dishBlack = UIColor(rgb: 0x010101)
+    static let dishLightGrey = UIColor(rgb: 0x828282)
+    static let dishGrey = UIColor(rgb: 0x4F4F4F)
+    static let dishWhite = UIColor(rgb: 0xFFFFFF)
+    static let dishBlue = UIColor(rgb: 0x0066D6)
+    static let dishSkyBlue = UIColor(rgb: 0x80BCFF)
+    
+    convenience init(red: Int, green: Int, blue: Int) {
+        assert(red >= 0 && red <= 255, "Invalid red component")
+        assert(green >= 0 && green <= 255, "Invalid green component")
+        assert(blue >= 0 && blue <= 255, "Invalid blue component")
+        
+        self.init(red: CGFloat(red) / 255.0, green: CGFloat(green) / 255.0, blue: CGFloat(blue) / 255.0, alpha: 1.0)
+    }
```

Comment:
초기값을 이용해서 alpha 값도 넘겨받을 수 있으면 더 좋을 것 같네요! 😊
```suggestion
    convenience init(red: Int, green: Int, blue: Int, alpha: CGFloat = 1.0) {
        assert(red >= 0 && red <= 255, "Invalid red component")
        assert(green >= 0 && green <= 255, "Invalid green component")
        assert(blue >= 0 && blue <= 255, "Invalid blue component")
        
        self.init(red: CGFloat(red) / 255.0, green: CGFloat(green) / 255.0, blue: CGFloat(blue) / 255.0, alpha: alpha)
    }
```

Reply:
넵 수정해보겠습니다!